physionet-django/events/forms.py

```diff
@@ -23,12 +23,21 @@ def __init__(self, user, *args, **kwargs):
         self.host = user
         super(AddEventForm, self).__init__(*args, **kwargs)
 
+    def clean(self):
+        cleaned_data = super(AddEventForm, self).clean()
+        if Event.objects.filter(title=cleaned_data['title'], host=self.host).exists():
```

Comment:
this clause raises an error if the title matches the event being edited. this shouldn't happen. 

i.e. currently if i click "edit event", i cannot save the the content unless i change the title.

Reply:
Ah thats a very good point. Let me fix that,  Good catch